tests/fox_consumer_tests.py

```diff
@@ -1,16 +1,46 @@
-from nose.tools import *
 from fox_consumer import Consumer
+from fox_consumer.consumer_db_support import PostgresWrapper
+from mock import patch
+from StringIO import StringIO
+from fox_consumer import FoxItem
+import unittest
+import json
+import sys
 
-def setup():
-    print "SETUP!"
+class TestFoxConsumer(unittest.TestCase):
+    @patch('fox_consumer.PostgresWrapper')          # 'override' the entire object
```

Comment:
because I don't need to test anything about the PostgresWrapper.
(the PostgresWrapper is just another attribute in the Consumer class...)